Docs/developer_guide/build_instructions/macos.md

```diff
@@ -5,113 +5,174 @@
 The prerequisites listed below are required to be able to configure/build/package/test Slicer.
 
 - XCode command line tools must be installed:
-```
+
+```bash
 xcode-select --install
 ```
+
 - A CMake version that meets at least the minimum required CMake version [here](https://github.com/Slicer/Slicer/blob/master/CMakeLists.txt#L1)
-- Large File Storage for git is required. (`brew install git-lvs`)
 - Qt 5: **tested and recommended**.
   - For building Slicer: download and execute [qt-unified-mac-x64-online.dmg](https://download.qt.io/official_releases/online_installers/qt-unified-mac-x64-online.dmg), install Qt 5.15, make sure to select `qtscript` and `qtwebengine` components.
   - For packaging and redistributing Slicer: build Qt using [qt-easy-build](https://github.com/jcfr/qt-easy-build#readme)
-- Setting `CMAKE_OSX_DEPLOYMENT_TARGET` CMake variable specifies the minimum macOS version a generated installer may target.  So it should be equal to or less than the version of SDK you are building on. Note that the SDK version is set using `CMAKE_OSX_SYSROOT` CMake variable automatically initialized during CMake configuration.
+- Setting `CMAKE_OSX_DEPLOYMENT_TARGET` CMake variable specifies the minimum macOS version a generated installer may target. So it should be equal to or less than the version of SDK you are building on. Note that the SDK version is set using `CMAKE_OSX_SYSROOT` CMake variable automatically initialized during CMake configuration.
 
 ## Checkout Slicer source files
 
 Notes:
+
 - While it is not enforced, we strongly recommend you to *avoid* the use of *spaces* for both the `source directory` and the `build directory`.
-- Due to maximum path length limitations during build the build process, source and build folders must be located in a folder with very short total path length. This is expecially critical on Windows and macOS. For example, `/sq5` has been confirmed to work on macOS.
+- Due to maximum path length limitations during build the build process, build folders must be located in a folder with very short total path length. This is expecially critical on Windows and macOS. For example, `/opt/s` has been confirmed to work on macOS.
 
 Check out the code using `git`:
-- Clone the github repository</p>
-```cd MyProjects
+
+- Clone the github repository
+
+```bash
+cd MyProjects
```

Comment:
Above it is written that source folder length matters, too. What is the recommended source folder location then?

Reply:
I've never seen a problem with the length of the source path.  I always use /Users/pieper/slicer/latest/Slicer with no problem.

Reply:
The source location is irrelevant on macOS. Should there be a recommended source location?

Reply:
Above where it talks about the length being important we could clarify that they source path doesn't matter, only the build path.